.gitignore

```diff
@@ -69,7 +69,7 @@ typings/
 .yarn-integrity
 
 # dotenv environment variables file
-.env
+#.env
```

Comment:
uncomment this, I didn't know that this was already in the gitignore

Reply:
done!